sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -91,38 +95,85 @@ public String getBlobServiceUrl() {
     /**
      * Creates a new container within a storage account. If a container with the same name already exists, the operation
      * fails.
+     * For more information, see the <a href="https://docs.microsoft.com/rest/api/storageservices/create-container">Azure Docs</a>.
      *
      * @param containerName The container name.
      */
-    public Void createContainer(String containerName) {
-        return storageBlobServiceClient.createContainer(containerName);
+    @NonNull
```

Comment:
is Void technically non-null? I'm not sure

Reply:
I'm pretty sure the only value a Void variable can have is null, so I wouldn't consider annotating it with `@NonNull`.